action.yml

```diff
@@ -1,5 +1,5 @@
 name: GitHub Script
```

Comment:
Do we also need 
Do we also need 
```suggestion
name: GitHub Script
author: 'GitHub'
```
here?

Reply:
Should we unquote GitHub for consistency?